langchain4j-mongodb-atlas/src/test/java/dev/langchain4j/store/embedding/mongodb/MongoDbEmbeddingStoreIT.java

```diff
@@ -0,0 +1,59 @@
+package dev.langchain4j.store.embedding.mongodb;
+
+import com.mongodb.client.MongoClient;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreIT;
+import dev.langchain4j.store.embedding.EmbeddingStoreWithFilteringIT;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.mongodb.MongoDBAtlasLocalContainer;
+
+import static dev.langchain4j.store.embedding.TestUtils.awaitUntilAsserted;
+import static dev.langchain4j.store.embedding.mongodb.MongoDbTestFixture.*;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+
+public class MongoDbEmbeddingStoreIT extends EmbeddingStoreIT {
```

Comment:
Let's rename it to `MongoDbEmbeddingStoreTest`, so it is executed on every build in [CI](https://github.com/langchain4j/langchain4j/actions/runs/11446855659/job/31846933655)

Reply:
I left a comment regarding renaming [here](https://github.com/langchain4j/langchain4j/pull/1932#discussion_r1805103259). I am not sure that it is a good idea to include this in every build, since the container startup time is long, even if it is done only once. I leave this to others to decide.

If the change should be made, I think it can be made in a separate PR.